src/SfxWeb/src/app/views/node/events/events.component.ts

```diff
@@ -13,16 +14,26 @@ export class EventsComponent extends NodeBaseControllerDirective {
 
   listEventStoreData: IEventStoreData<any, any> [];
   optionsConfig: IOptionConfig;
+  @Input() isShown = true;
 
-  constructor(protected data: DataService, injector: Injector) {
+  constructor(protected data: DataService, injector: Injector, private settings: SettingsService) {
     super(data, injector);
   }
 
   setup() {
     this.listEventStoreData = [
-      this.data.getNodeEventData(this.nodeName)
+      this.data.getNodeEventData(this.nodeName),
+      this.data.getClusterEventData(),
     ];
 
+    this.data.clusterManifest.ensureInitialized().subscribe(() => {
```

Comment:
is this necessary? I thought that the options config for enable repair tasks should handle it? is that not the case?

Reply:
this would be a good time to refactor this to a common spot, like the data service because it looks like originally this was just on the cluster event's page.